frontends/election-manager/src/utils/election.ts

```diff
@@ -264,3 +264,45 @@ export function generateBlankBallots({
 
   return ballots;
 }
+
+export function generateOvervoteBallot({
+  election,
+  precinctId,
+}: {
+  election: Election;
+  precinctId: PrecinctId;
+}): Dictionary<string | VotesDict> | undefined {
+  const precinctBallotStyles = election.ballotStyles.filter((bs) =>
+    bs.precincts.includes(precinctId)
+  );
+
+  const ballot: Dictionary<string | VotesDict> = { precinctId };
+  const votes: VotesDict = {};
+
+  for (const ballotStyle of precinctBallotStyles) {
+    const contests = election.contests.filter(
+      (c) =>
+        ballotStyle.districts.includes(c.districtId) &&
+        ballotStyle.partyId === c.partyId
+    );
+
+    for (const contest of contests) {
```

Comment:
While it's reasonably clear reading the code that we're generating a minimally overvoted ballot, worth adding a comment noting that! Maybe with a (super brief) explanation as to why we made that call